config/webhook/kustomizeconfig.yaml

```diff
@@ -0,0 +1,25 @@
+# the following config is for teaching kustomize where to look at when substituting vars.
+# It requires kustomize v2.1.0 or newer to work properly.
+nameReference:
+- kind: Service
+  version: v1
+  fieldSpecs:
+  - kind: MutatingWebhookConfiguration
+    group: admissionregistration.k8s.io
+    path: webhooks/clientConfig/service/name
```

Comment:
Are these stanzas for a mutating webhook needed even though we only actually have a validating webhook?

Reply:
These were generated by kubebuilder.